tests/roles/neutron_adoption/tasks/main.yaml

```diff
@@ -0,0 +1,35 @@
+- name: deploy podified Neutron
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc patch openstackcontrolplane openstack --type=merge --patch '
+    spec:
+      neutron:
+        enabled: true
+        template:
+          databaseInstance: openstack
+          secret: osp-secret
+    '
+
+- name: wait for Neutron to start up
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc get pod --selector=service=neutron -o jsonpath='{.items[0].status.phase}{"\n"}' | grep Running
+  register: neutron_running_result
+  until: neutron_running_result is success
+  retries: 60
+  delay: 2
+
+- name: check that Neutron is reachable and its endpoints are defined
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    alias openstack="oc exec -t openstackclient -- openstack"
+
+    openstack endpoint list | grep network
+    openstack network list
+  register: neutron_responding_result
+  until: neutron_responding_result is success
+  retries: 15
+  delay: 2
```

Comment:
late to the party but looks like this is running in the ci job successfully

https://logserver.rdoproject.org/75/75/2c2b5544ffd85946282f4bf147e036f48ad44afe/github-check/centos-9-standalone-to-crc-data-plane-adoption/dcb4f51/controller/data-plane-adoption-tests-repo/data-plane-adoption/tests/logs/test_minimal_out_2023-08-02T16:20:16EDT.log

TASK [neutron_adoption : check that Neutron is reachable and its endpoints are defined] ***


the other job (single node) failed earlier on cinder/unrelated issue

Reply:
also sanity checked we have neutron in the kustomization container overrides template for the periodic job https://github.com/openstack-k8s-operators/data-plane-adoption/blob/58909ebdc2a6c7c4217ae758d8563efa9e52aeee/tests/roles/backend_services/templates/container_overrides.j2#L54